descartes_utilities/package.xml

```diff
@@ -11,6 +11,7 @@
   <maintainer email="Jmeyer@swri.org">Jonathan Meyer</maintainer>
 
   <license>Apache 2.0</license>
+  <license>BSD</license>
```

Comment:
I lifted some code from the ecl library, which is licensed as BSD. I imagine I should include the BSD clause in that source file? I have no idea how this dual licensing stuff would work.